frontend/packages/helm-plugin/src/components/forms/install-upgrade/HelmInstallUpgradePage.tsx

```diff
@@ -74,7 +76,7 @@ const HelmInstallUpgradePage: React.FunctionComponent<HelmInstallUpgradePageProp
   const [initialFormSchema, setInitialFormSchema] = React.useState<JSONSchema7>();
 
   const helmAction: HelmActionType =
-    initialChartURL !== 'null' ? HelmActionType.Install : HelmActionType.Upgrade;
+    initialChartURL !== null ? HelmActionType.Install : HelmActionType.Upgrade;
```

Comment:
Can you explain why this is now `null` instead of `"null"`? And can we simplify this?

```suggestion
    !initialChartURL ? HelmActionType.Install : HelmActionType.Upgrade;
```

Reply:
Thanks for explaining this :+1: